src/components/views/rooms/WhoIsTypingTile.tsx

```diff
@@ -74,34 +82,36 @@ export default class WhoIsTypingTile extends React.Component {
         Object.values(this.state.delayedStopTypingTimers).forEach((t) => t.abort());
     }
 
-    _isVisible(state) {
+    _isVisible(state: IState): boolean {
```

Comment:
please mark as private and remove the prefix

Reply:
Can't remove the prefix as it will clash with another field